src/index.html

```diff
@@ -7,6 +7,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
import it directly inside stylesheets

Reply:
for now it can be here ;)

Reply:
If you don't change it now you will not change it later - trust me